hatch/models.py

```diff
@@ -41,9 +41,20 @@ def get_sha(path):
 def get_files(path):
     """List all files in a directory recursively as a flat list.
 
-    Sorted, and does not include directory entries.
+    Sorted, and excluding various files
     """
-    return list(sorted([p.relative_to(path) for p in path.glob("**/*") if p.is_file()]))
+
+    def exclude(p):
+        strp = str(p)
```

Comment:
`strp` made me think this was related to `strptime`.  Maybe `path` is fine here? (I realise it's not a Path instance but I think that's fine given we treat it like a string belod)